internal/cephfs/volume.go

```diff
@@ -30,14 +30,13 @@ import (
 )
 
 var (
-	// cephfsInit is used to create "csi" subvolume group for the first time the csi plugin loads.
-	// Subvolume group create gets called every time the plugin loads, though it doesn't result in error
-	// its unnecessary
-	cephfsInit = false
+	// checkAdditionalInfo stores the mapping for clusterID and resize is
+	// supported in the particular cluster and whether subvolumegroup is
+	// created or not.
```

Comment:
may be the requirement of knowing about these additional information can be put in the source comment? why or where its useful?

Reply:
> may be the requirement of knowing about these additional information can be put in the source comment? why or where its useful?

Thanks. I'll add it.